docs/index.rst

```diff
@@ -24,8 +23,8 @@ Using Pyodide
 =============
 
 Pyodide may be used in several ways: directly from JavaScript, or to execute
-Python scripts asynchronously in a web worker. Although still experimental,
-additional packages may be installed from PyPI to be used with Pyodide.
+Python scripts asynchronously in a web worker. Additional packages may be
```

Comment:
micropip is as experimental as everything else in pyodiode, I'm not sure we need to highlight it specifically.

Reply:
This version sounds much better. Maybe we could also name it

> Additional pure Python packages may be installed from PyPI to be used with Pyodide.

Reply:
> Including might be a bit too long in the title, also I'm not sure what's the correct name for this grammatical construction but the two commas make it a bit harder to read IMO. How about,
> 
> > Python with the scientific stack, compiled to WebAssembly
> 
> It might be a bit better than "and"

That is nice. What about [this comment](https://github.com/iodide-project/pyodide/pull/1246#discussion_r575882456)?